app/src/main/res/drawable/btn_rounded_corner.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would suggest to not use short words, example button instead of btn. Also you are creating a background and not a button. Maybe use background_rounded_corner or background_blue_rounded_corner